src/nifgen/metadata/enums_addon.py

```diff
@@ -8,4 +8,97 @@
 enums_codegen_method = {
 }
 
-
+enums_additional_enums = {
+    'RelativeTo': {
+        'values': [
+            {
+                'name': 'NIFGEN_VAL_WAVEFORM_POSITION_START',
+                'value': 0,
+            },
+            {
+                'name': 'NIFGEN_VAL_WAVEFORM_POSITION_CURRENT',
+                'value': 1,
+            },
+        ],
+    },
+    'TriggerWhen': {
+        'values': [
+            {
+                'name': 'HighLevel',
```

Comment:
Use name 'NIFGEN_VAL_ACTIVE_HIGH'

Reply:
In other words, all names should be identical to the names used by the C API.